src/components/index.ts

```diff
@@ -8,6 +8,7 @@ export {
   DropdownItemProps,
 } from './Dropdown/Dropdown';
 export { TextInput, TextInputProps } from './Form/TextInput';
+export { Checkbox, CheckboxProps } from './Form/Checkbox';
```

Comment:
Should this also export the `CheckboxInputProps`?

Reply:
Not sure if that's necessary. The interface is used to specify which attributes are available to be passed to the input element when using the `checkboxInputProps` prop of the parent component.

Reply:
I thing it's more a matter of consistency. If we have exported sub-types for the API elsewhere then we should do it here as well. It is also a matter of architecture, if we would be using composition then it would not be as necessary. With children as props it's more useful to provide the typings since w might want to do `const childProps: ChildProps = ...` and `<Comp childProps={childProps} />` if our code style so requires.